css/my_portfolio.css

```diff
@@ -0,0 +1,95 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+  background: #e5e5e5;
+}
+
+.list-style-none {
+  list-style: none;
+}
+
+.d-none {
+  display: none;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.d-block {
+  display: block;
+}
+
+.d-grid {
+  display: grid;
+}
+
+.margin-left-5 {
+  margin-left: 5%;
+}
+
+.pos-relative {
+  position: relative;
+}
+
+.pos-absolute {
+  position: absolute;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  width: 100%;
+  background: #3c3a39;
+  height: 4em;
+  padding: 10%;
+}
+
+.logo {
+  color: #fff;
+}
+
+.my-self {
+  background: #1c1a19;
+  padding: 35% 5%;
+  height: 100vh;
+}
+
+.myself-header {
+  width: 95%;
+}
+
+.my-name {
+  font-family: 'Poppins', sans-serif;
+  color: #ff6b00;
+  font-size: 37px;
+  width: 88%;
+  line-height: 64px;
+}
+
+.my-work {
+  color: #fff5e1;
+}
+
+.my-text {
+  color: #fff;
+  margin: 5% 0;
+  font-family: 'Poppins', sans-serif;
+  line-height: 24px;
+}
+
+.group1 {
+  background-image: url("../assets/images/group1.svg");
```

Comment:
- [ ] Very good of you to include the matching background image from the Figma 👏. Please ensure that the positioning for the background image is the same as the Figma design.
## Your Design
![image](https://user-images.githubusercontent.com/18730106/143314036-45a8f7a5-113a-4d79-a9bd-7196c2fd96d0.png)
If you can see on the Figma the background image aligns perfectly to the right of the screen and it doesn't cover any of the headline text:
## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/143314304-bc7e8d6e-c224-4214-9c14-775519686fb3.png)

Please implement these changes as it is to help you with your understanding of how to match your CSS to a set template. Sometimes you will need to match an already provided template and they will want it to match exactly the design 🙏. Aside from that, your design looks really good.